src/rank_llm/retrieve/retriever.py

```diff
@@ -172,7 +172,7 @@ def retrieve(self) -> List[Dict[str, Any]]:
                         f"Invalid dataset format: {self._dataset}. Expected a list of strings where each string represents a document."
                     )
                 document_hits.append(
-                    {"content": document, "qid": 1, "docid": i + 1, "rank": i + 1}
+                    {"content": document, "qid": 1, "docid": i + 1, "rank": i + 1, "score": 0}

```

Comment:
This only applies to the inline_docs demo; writing results in trec eval format would throw an exception since the results do not have a "score" key. I put a placeholder here for now to resolve the bug, but please let me know how you would like this to be handled.

Reply:
can we use rank instead?